cmake/std/pr_config/pullrequest.ini

```diff
@@ -267,6 +268,19 @@ setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
 
 
 
+[Trilinos_pullrequest_intel_19.0.5]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 6.1.0
+module-load sems-intel: 19.0.5
+module-load sems-mpich: 3.2
+use SEMS-DEFAULT:
+module-remove sems-openmpi:
+use ATDM-DEFAULT:
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
```

Comment:
Why are we forcing the local compiler version to 4.9.3? Why not something newer? 4.9.3 doesn't support C++14.

Reply:
@jwillenbring I was following the pattern in most all the other builds. 4.9.3 is used for force local compiler in GCC 7.2.0, 7.2 debug, 7.2 serial, 8.3.0 & Intel 17.0.1. The only ones that push higher are the Clang builds.
But going to 5.x for C++14 support or higher to match the build makes complete sense to me.